File: utils/handler.go
package utils

import (
	"PluginEngine/constants"
)

func ErrorHandler(errorCode string, errMessage string) map[string]interface{} {

	error := make(map[string]interface{})

	error[constants.ErrorCode] = errorCode

	error[constants.ErrorMessage] = errMessage

	return error

}

File: Motadata/src/main/java/org/motadata/util/Handler.java
package org.motadata.util;

import io.vertx.core.json.JsonObject;
import org.motadata.constants.Constants;

public class Handler {

   public static JsonObject errorHandler(String error, String errorMessage, String errorCode)
    {
        JsonObject result = new JsonObject();

        result.put(Constants.ERROR,error);

        result.put(Constants.ERROR_MESSAGE,errorMessage);

        result.put(Constants.ERROR_CODE,errorCode);

        result.put(Constants.STATUS,Constants.FAIL);

        return result;
    }
}
